src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,76 @@
+import classNames from 'classnames';
+
+interface TodoFooterProps {
+  remainingTodos: number;
+  filter: string;
+  setFilter: (filter: string) => void;
+  hasCompletedTodos: boolean;
+  onClearCompleted: () => void;
+}
+
+export const TodoFooter: React.FC<TodoFooterProps> = (
+  {
+    remainingTodos,
+    filter,
+    setFilter,
+    hasCompletedTodos,
+    onClearCompleted,
+  },
+) => (
+  <footer className="todoapp__footer">
+    <span className="todo-count">
+      {remainingTodos}
+      {' items left'}
+    </span>
+
+    <nav className="filter">
+      <a
+        href="#/"
+        className={
+          classNames(
+            'filter__link',
+            { selected: filter === 'all' },
```

Comment:
You should use it in all places where it occurs
```suggestion
            { selected: filter === FilterType.ALL },
```